stdlib/public/TensorFlow/NumpyConversion.swift

```diff
@@ -0,0 +1,99 @@
+//===-- NumPyConversion.swift ---------------------------------*- swift -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines conversion initializers from `numpy.ndarray` to
+// `ShapedArray` and `Tensor`.
+//
+//===----------------------------------------------------------------------===//
+
+#if canImport(Python)
+import Python
+
+extension ShapedArray : NumpyArrayConvertible
+  where Scalar : NumpyScalarCompatible {
+  public init?(numpyArray: PyValue) {
+    guard let np = try? Python.attemptImport("numpy") else { return nil }
```

Comment:
See comment above. Use a fileprivate global variable to represent a loaded numpy module.